temporal-server.yaml

```diff
@@ -59,6 +59,19 @@ subpackages:
           install -Dm755 temporal-sql-tool "${{targets.subpkgdir}}"/usr/bin/temporal-sql-tool
       - uses: strip
 
+  - name: temporal-server-oci-entrypoint
+    description: Entrypoint for using temporal server in OCI containers
+    dependencies:
+      runtime:
+        - bash
+        - dockerize
```

Comment:
Let's drop this and have the image add it back to match https://github.com/wolfi-dev/os/pull/8565

```suggestion
```

Reply:
Done, thanks for the suggestion @jonjohnsonjr ❤️